platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -93,8 +93,8 @@ extension PolyOut {
                         acc.isEmpty ? x : acc + "," + x
                     }
                 }
-                completionHandler(FileStats(
-                    isDirectory: isDir.boolValue,
+                completionHandler(Stats(
+                    directory: isDir.boolValue,
```

Comment:
I rather liked the original name, it's much more clear. As long as we're mapping from the original in `Stats`, why not keep it?

Reply:
Because that's not what the property is called, it's called `directory`. Again, not gonna change the API while doing a fix.